website/docs/framework-comparison.md

```diff
@@ -134,3 +136,34 @@ val schema: GraphQLSchema = toSchema(config, queries)
 ```
 
 This makes changes in code directly reflect to your schema and you can still produce the `GraphQLSchema` to print and export an SDL file.
+
+
+## DGS
+[DGS](https://netflix.github.io/dgs/) is a GraphQL server framework for Spring Boot. It works with both Java and Kotlin.
+DGS is also built on top of `graphql-java` and implements many similar features to `graphql-kotlin` and [graphql-java-kickstart/graphql-spring-boot](https://github.com/graphql-java-kickstart/graphql-spring-boot).
+
+* Auto-configuration of server routes and request handling
+* Auto-wiring of data fetchers (resolvers) to the `GraphQLSchema`
+* Apollo Federation support
+* Subscriptions support
+* Client schema-code generation
+
+While both libraries do very similar things, there are some minor differences which may serve different usecases better.
+As with open source library, you can compare and use the right tool for the job.
+
+### Extra Features of DGS
+
+* Support for a SDL-First (Schema-First) approach
+* Included [JsonPath](https://github.com/json-path/JsonPath) testing library
+* Included schema response mocking components
+* Uses Spring MVC
+
+### Extra Features of graphql-kotlin
+
+* Ability to use any server framework outside of Spring, like Ktor
```

Comment:
* Supports code-first approach (generates schema from source code - does not require duplicate implementation of data fetchers, schema classes, and SDL files)
* Reference server implementation build on top of [Spring Webflux]((https://spring.io/reactive) for a reactive server stack
* Abstract server logic can be used in any framework, e.g. Ktor